GANDLF/compute/forward_pass.py

```diff
@@ -51,21 +51,22 @@ def validate_network(
     print("*" * 20)
     # Initialize a few things
     total_epoch_valid_loss = 0
-    total_epoch_valid_metric = {}
+    total_epoch_valid_metric: dict[str, Union[float, np.array]] = {}
     average_epoch_valid_metric = {}
 
     for metric in params["metrics"]:
         if "per_label" in metric:
-            total_epoch_valid_metric[metric] = []
+            total_epoch_valid_metric[metric] = np.zeros(
+                shape=params["model"]["num_classes"]
+            )
```

Comment:
Just a beautification: when we start to use this field, we immediately replace list with np.array and start to sum. So, simplified code  here https://github.com/mlcommons/GaNDLF/pull/868/files#diff-9930b6bd597da7670c42f9dc94d740c1d3f98533c0422682061e023d3b73bb38L214-R215 not to worry about empty lists & init (just sum results as usual from the scratch)